### DISCUSSION TITLE
Remapping Assistance Needed.

### DISCUSSION THREAD

**Discussion by Zonark** [darthcloud/BlueRetro#440] [Q&A]
I have attempted to use the advance config via the web ui linked under the section but I genuinely have no clue what I am looking at.
What I am trying to do is Map the Xbox Series X controller to Gamecube what I want to change is the ABXY to A=A B=B X=X Y=Y

Right now its default which I think is A=A B+X X+B Y+Y. 

I have chrome download an I am using it to connect as recommended.

Is there any external applications to assist with this or could someone walk me through the correct steps. Thank you in advance

**Comment by vicsidious** [comment]
Hey Zonark, 
Here's what you gotta do: 
To help you put the correct labels on the universal inputs you see when you load up the page, you have to select what is your source and destination! "Source" being your best approximation of the controller used  (Here Xbox) and "Destination", being the system (Gamecube in this scenario). Once it's done you can see something a little more familiar, like this: 
![image](https://user-images.githubusercontent.com/115555660/197611262-e05c624f-2ca3-4920-8fbb-e004337a839a.png)
Now, to achieve what you wish to do, you have to swap what's in the red rectangle, the "B" and "X" selections must be swapped in one column or the other, but obviously not both... 

Keep in mind that this change will affect every system and every controller you use. If you wish to change controller and revert back to the default settings, you can change it back manually or apply the "Default Preset", in the preset section.
